libs/platform/src/lib/table/table.component.scss

```diff
@@ -194,9 +194,12 @@ $fd-table-scrollbar-width-with-border: calc(#{$fd-table-scrollbar-width} + var(-
         .fd-table__cell {
             margin-top: 0;
             max-height: 100%;
-            position: sticky;
             top: 0;
 
+            &--fixed {
```

Comment:
Platform docs example?

Reply:
Coming in a later PR when addressing #9030